### ISSUE TITLE
The Connect4 game no longer works :(

### ISSUE THREAD

**Issue by sdennett55** [tannerkrewson/lansite#1]
Looks like you have a more polished version here: https://kevinshannon.dev/connect4

Error: 
```
Uncaught ReferenceError: Peer is not defined
    at setUpOnline (connect4.js:1027)
```

Love this app, btw. Exactly what I was looking for. Having fun making some altercations.

**Comment by tannerkrewson** [comment]
Thanks! 😄 

@kevin-shannon and I removed the api that Lansite uses to interface with connect4 in a redesign. I had a fork of the connect4 version at tannerkrewson/connect4lansite, but i think i deleted that. We've since implemented a new system (https://github.com/kevin-shannon/connect4/issues/25) for connect4 multiplayer that might work. It's been awhile anyways so I'll take a look.

**Comment by tannerkrewson** [comment]
fixed with 37745ca01a9070aebce2fd298c4f45d545e7cc75

lmk if it works :)

**Comment by sdennett55** [comment]
> fixed with [37745ca](https://github.com/tannerkrewson/lansite/commit/37745ca01a9070aebce2fd298c4f45d545e7cc75)
> 
> lmk if it works :)

wow, you are the best! works great! thanks for taking the time to dig into this!

On another note, been playing around with a few things:
- auto assigning colors to users
- adding steam profile pics and timestamps to messages
- adding option to allow all users to be admin by default
- allowing the Match Finder, Connect4 player, and an added Introductory TextBox to be pinned to the top by default 
- adding another info section for the Discord link and WiFi info. If you're interested I can look into creating a pull request for one or more of these features.

![duxlan2](https://user-images.githubusercontent.com/5250779/71550928-6b9ec300-29a9-11ea-845b-dadee44d6712.PNG)

**Comment by tannerkrewson** [comment]
Wow those are amazing ideas. i would be honored to merge them into the main repo! 😄
